src/test/ui/rfc-2008-non-exhaustive/enum_same_crate_empty_match.rs

```diff
@@ -25,7 +25,7 @@ pub enum EmptyNonExhaustiveEnum {}
 fn empty_non_exhaustive(x: EmptyNonExhaustiveEnum) {
     match x {}
     match x {
-        _ => {} // not detected as unreachable
+        _ => {}, //~ ERROR unreachable pattern
```

Comment:
Is there a reason the comma was added?

Reply:
No, I just copy-pasted the line from another file

Reply:
```suggestion
        _ => {} //~ ERROR unreachable pattern
```